plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/order-by-control/default-query-order-by-control.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { store as coreStore } from '@wordpress/core-data';
+import { dispatch, select } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import OrderByControl from './order-by-control';
+
+const orderOptions = [
+	{
+		label: __( 'Newest to oldest', 'woocommerce' ),
+		value: 'date',
+	},
+	{
+		label: __( 'Price, high to low', 'woocommerce' ),
+		value: 'price-desc',
+	},
+	{
+		label: __( 'Price, low to high', 'woocommerce' ),
+		value: 'price',
+	},
+	{
+		label: __( 'Sales, high to low', 'woocommerce' ),
+		value: 'popularity',
+	},
+	{
+		label: __( 'Rating, high to low', 'woocommerce' ),
+		value: 'rating',
+	},
+	{
+		// In WooCommerce, "Manual (menu order)" refers to a custom ordering set by the store owner.
+		// Products can be manually arranged in the desired order in the WooCommerce admin panel.
+		value: 'menu_order',
+		label: __( 'Manual (menu order)', 'woocommerce' ),
+	},
+];
+
+const DefaultQueryOrderByControl = () => {
+	const settings = select( 'core' ).getEditedEntityRecord(
+		'root',
+		'site'
+	) as Record< string, string >;
+
+	const [ value, setValue ] = useState(
+		settings.woocommerce_default_catalog_orderby || 'menu_order'
+	);
+
+	const onChange = ( newValue: string ) => {
+		setValue( newValue );
+		dispatch( coreStore ).editEntityRecord( 'root', 'site', undefined, {
+			[ `woocommerce_default_catalog_orderby` ]: newValue,
+		} );
+	};
```

Comment:
I think we could track this as well to see the usage among other filters:

```suggestion
	const onChange = ( newValue: string ) => {
		setValue( newValue );
		dispatch( coreStore ).editEntityRecord( 'root', 'site', undefined, {
			[ `woocommerce_default_catalog_orderby` ]: newValue,
		} );
		trackInteraction( CoreFilterNames.DEFAULT_ORDER );
	};
```

You'd have to define new `CoreFilterNames`. WDYT?

Reply:
Sounds good! Done in b06a85ec260739cc335888a18e006d3c66dada1d.